### ISSUE TITLE
New Project idea : Emotion Detection

### ISSUE THREAD

**Issue by AM1CODES** [akshitagupta15june/Face-X#373]
**Is your feature request related to a problem? Please describe.**
The new project idea involves detecting human emotions, these emotions can be used to build a recommender that would recommend songs/movies by detecting the real time emotions of user or maybe used for other purpose similar to this.

**Describe the solution you'd like**
 I would like to work on building the emotion detector for detecting human emotions

**Describe alternatives you've considered**
Instead of just working on the model, i would also like to turn it into a GUI application using Python and Tkinter.

**Approach to be followed (optional)**
* Model creation using Tensorflow and Keras.
* Exporting this model to some format such as .h5 file.
* Using opencv for face detection.
* Using the model to predict human emotions

**Comment by akshitagupta15june** [comment]
hi here you can see what is already implemented till now and let me know what new you want to add https://github.com/akshitagupta15june/Face-X/tree/master/Face-Emotions-Recognition

**Comment by AM1CODES** [comment]
@akshitagupta15june I went through the content of the folder and i think all of these are implemented well already. I too implemented it using custom CNN.  The performance is as follows - 
* Training accuracy - 71.73
* Testing accuracy - 65..22
One of the models in the folder has an accuracy of 64% on testing set which i think is good given the data we had. So i think there are sufficient models in the repo. So if you wish to close this issue you can, but if you want, i can add my model as well and just enclose it in a GUI with some buttons and instructions. 

Also this repository is very useful and informative so thank you for creating it and maintaining it so well!

**Comment by akshitagupta15june** [comment]
You can add you your model inside the folder of emotions one and also follow the naming convention as emotion-recognition-with-GUI

**Comment by AM1CODES** [comment]
Okay! I have started working on it. Thank you.
